docs/6.x/pack.md

```diff
@@ -805,6 +811,33 @@ correct image references.
     There is a sample application available on [GitHub](https://github.com/gravitational/quickstart/tree/master/mattermost)
     that demonstrates this workflow.
 
+### Customizing Helm values
+
+It is possible to customize values of your Helm charts when installing or
+upgrading the application. To provide custom Helm values at install time,
+pass them via `--values` and `--set` flags to `gravity install` command:
+
+```bash
+unpacked-image$ ./gravity install --values=custom-values.yaml --set=nginx.image=1.9.1 --set=postgres.registry=internal.registry.io
```

Comment:
Hmmm, doesn't this imply that the --set will get applied to every helm chart contained within the cluster image? What happens if multiple charts have overlapping values?

Reply:
Not really, all these values (provided by values/set flags) will be merged into a single values.yaml file and how they are applied really depends on the command that runs in the hook - we don't apply those values automatically.

So for multiple charts with possibly overlapping values, the user would need to create appropriate nesting for chart specific values, which is I think a regular practice in the Helm world anyway, for example:

```yaml
postgres: // values for chart 1
  image: 12.0.0
mariadb: // values for chart 2
  image: 11.2.3
```

and then when running helm install/upgrade, the appropriate values would get applied to the appropriate charts.